apps/runLmdFit.cxx

```diff
@@ -31,9 +31,9 @@ void runLmdFit(string input_file_dir, string fit_config_path, string acceptance_
   lmd_runtime_config.setAcceptanceResolutionInputDirectory(acceptance_file_dir);
   lmd_runtime_config.setReferenceAcceptanceResolutionInputDirectory(reference_acceptance_file_dir);
 
-  lmd_runtime_config.setElasticDataName("lmd_data_.*of.*.root");
-  lmd_runtime_config.setAccDataName("lmd_acc_data_.*of1.root");
-  lmd_runtime_config.setResDataName("lmd_res_data_.*of1.root");
+  lmd_runtime_config.setElasticDataName("lmd_data.root");
+  lmd_runtime_config.setAccDataName("lmd_acc_data.root");
+  lmd_runtime_config.setResDataName("lmd_res_data.root");
```

Comment:
Either change the values back, because the cluster scripts generate these output names. Or even better add some options that can modify these filenames, while the default are set to those `...of...` values